### DISCUSSION TITLE
ExSys EX-1113HMS not working

### DISCUSSION THREAD

**Discussion by a-kipp** [mvp/uhubctl#427] [Q&A]
Hi,

I'm trying to get power-port switching running on ExSys EX-1113HMS. It is shown as a supported device when running the uhubctl command. It has 16 physical ports and appears as 5 hubs where 4 hubs a connected to one top-level hub. When running for example: ```uhubctl -a toggle -l 1-5.1.1``` I get: "No compatible devices detected at location 1-5.1.1!". However I can do ```uhubctl -a toggle -l 1-5.1``` to address the top-level hub, in that case it shows that the port is turned off, but in actuality still all ports are powered.

**Comment by mvp** [comment]
Please show full output of `sudo uhubctl `

**Reply by a-kipp** [reply]
Thanks for the quick response,
turned out out I addressed the ports in in the wrong way. I had to do ```uhubctl -a toggle -l 1-5.1 -p 1``` instead of  ```uhubctl -a toggle -l 1-5.1.1```

**Comment by slk9** [comment]
I have tried ExSys EX-1113HMS today on MacOS. It does not work as expected.
Yes uhubctl will cut the connectivity so no communication from host to device connected and ports reported as off.
But all ports still provides voltage and current to devices connected. Which makes ExSys EX-1113HMS useless for most usecases when we need uhubctl.

**Reply by slk9** [reply]
ExSys EX-1113HMS HUB details:
Current status for hub 0-2.3.4 [05e3:0620 GenesysLogic USB3.2 Hub, USB 3.20, 4 ports, ppps]
  Port 1: 00a0 off
  Port 2: 00a0 off
  Port 3: 00a0 off
  Port 4: 00a0 off
Current status for hub 0-2.3.1 [05e3:0620 GenesysLogic USB3.2 Hub, USB 3.20, 4 ports, ppps]
  Port 1: 00a0 off
  Port 2: 00a0 off
  Port 3: 00a0 off
  Port 4: 00a0 off
